### PR TITLE
[website] Redesigned

### LINKED ISSUE #3015: [RFC] Redesign Homepage of website
References:
https://alpha.nativebase.io/
https://akveo.github.io/react-native-ui-kitten/ 

https://docusaurus.io/showcase/

@hetanthakkar1 @khushal87 @HariAcidReign @jugshaurya we can together work on this

### PR DESCRIPTION
**Pull request by arpitBhalla**
Closes #3015
Preview https://rne-docs.surge.sh/
### HomePage
scrnli_26_06_2021_18-27-36

### Tabs:
image

### Expo Snack: 
Screenshot from 2021-06-25 23-56-39

### Tips
image

### CODE REVIEW

**Comment by arpitBhalla** [comment]
image

Screenshot from 2021-06-25 23-13-23

**Comment by khushal87** [comment]
Make sure you don't change the directory structure of the docs as mentioned in this screenshot https://user-images.githubusercontent.com/55053424/123469721-0a44c800-d611-11eb-8b92-397ab1affba3.png for now. You can do it later.

**Comment by khushal87** [comment]
Hompage renovation is a great idea, but don't add additional restructuring of the docs in the same PR, it's difficult to track these changes and when it's merged, If you do it same PR. Please raise different PRs for different changes or do add a detailed description.

File: website/src/components/Features.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          <div className="row">
+            <div className="col col--10 col--offset-1">
+              <p className="p--desc margin-vert--md text--center">
+                UI Kitten is a React Native implementation of the Eva Design
+                System. The framework contains a set of general-purpose UI
+                components styled in a similar way. The most awesome thing about
+                Ui Kitten is that the themes can be changed in the runtime
+                without reloading the application. This way, you may easily
+                focus on business logic, while UI Kitten takes care of the
+                visual appearance of your product.
```
Comment (arpitBhalla) [inline website/src/components/Features.tsx:56]: 🙄

File: website/src/components/RunOnExpo.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <div className="col col--8 text--center">
+              <h1 className="hero__title gradient clip-text">The Demo App</h1>
+              <p className="p--desc">
+                To see UI Kitten in action, download a live demo published on
+                both App Store and Google Play or run it by cloning a GitHub
+                repo. You can use it as a starter kit for your next mobile app
+                for any domain: e-commerce, social, fitness, etc. Compose the
+                application from available screens, add backend integration, and
+                you will end up with an A-grade cross-platform mobile
+                application.
```
Comment (arpitBhalla) [inline website/src/components/RunOnExpo.tsx:23]: 🙄

**Comment by arpitBhalla** [comment]
> Make sure you don't change the directory structure of the docs as mentioned in this screenshot https://user-images.githubusercontent.com/55053424/123469721-0a44c800-d611-11eb-8b92-397ab1affba3.png for now. You can do it later.

Okay, I will do accordingly @khushal87

**Pull request merged**